Controller/SecurityController.php

```diff
@@ -53,7 +57,16 @@ public function loginAction(Request $request)
             ? $this->get('security.csrf.token_manager')->getToken('authenticate')->getValue()
             : null;
 
+         $form = $this->createFormBuilder()
+            ->add('_username', TextType::class, array('data' => $lastUsername))
+            ->add('_password', PasswordType::class)
+            ->add('_remember_me', CheckboxType::class, array('required' => false))
+            ->add('_submit', SubmitType::class)
```

Comment:
Yes, you mentioned in the other PR, I need to add a FormType to add compatibility, thanks!